core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -3779,7 +3780,7 @@ public long doCountUsersWithClaims(String claimUri, String value) throws UserSto
             if (log.isDebugEnabled()) {
                 log.debug(msg + sqlStmt);
             }
-            throw new UserStoreException(msg, e);
+            throw new UserStoreException(msg, ERROR_CODE_ERROR_WHILE_EXECUTING_THE_SQL.getCode(), e);
```

Comment:
%s is not replaced here. 

Anyway, don't append the SQL query.

Reply:
fixed